<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adsk::USDAssetResolverSettingsWidget</class>
 <widget class="QWidget" name="Adsk::USDAssetResolverSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>USD Asset Resolver Settings</string>
  </property>
  <layout class="QGridLayout" name="mainLayout" columnstretch="0,4">
   <item row="1" column="1">
    <widget class="QLineEdit" name="mappingFilePath"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Mapping File: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="includeProjectTokens">
     <property name="text">
      <string>Include Project Tokens</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save &amp;&amp; Refresh</string>
       </property>
       <property name="toolTip">
        <string>Save settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>